v03_pipeline/lib/tasks/update_variant_annotations_table_with_new_samples_test.py

```diff
@@ -1436,8 +1460,13 @@ def test_sv_update_vat(
             ],
         )
 
-    def test_gcnv_update_vat(self, mock_update_rdc_task: None) -> None:
+    def test_gcnv_update_vat(
+        self,
+        mock_update_rdc_task,
+        mock_update_vat_with_rdc_task,
```

Comment:
If we lift the mocking down from the class onto the individual functions we can exclude the mocks from SV and GCNV right?

Reply:
good point, I was in a copy paste frenzy but these mocks definitely don't go here